lib/ansible/modules/cloud/amazon/lambda.py

```diff
@@ -334,19 +346,24 @@ def main():
             func_kwargs.update({'Timeout': timeout})
         if memory_size and current_config['MemorySize'] != memory_size:
             func_kwargs.update({'MemorySize': memory_size})
-        if (environment_variables is not None) and (current_config.get('Environment', {}).get('Variables', {}) != environment_variables):
-            func_kwargs.update({'Environment':{'Variables': environment_variables}})
+        if (environment_variables is not None) and (current_config.get(
+                'Environment', {}).get('Variables', {}) != environment_variables):
+            func_kwargs.update(
```

Comment:
No need to change this from what it was. All of the other unnecessary line breaks following this that can be undone too.